typings/index.d.ts

```diff
@@ -46,15 +46,14 @@ export interface RenderResult extends GetsAndQueries {
   debug: () => void
   rerender: (ui: React.ReactElement<any>) => void
   unmount: VoidFunction
+  cleanup: VoidFunction
```

Comment:
Is this an established practice to have a single type definition for functions with different purposes? I would avoid sharing the type because these functions, although look similar, are not the same thing. For example, `debug` looks like a `VoidFunction` but the type is not shared. Oh by the way, I don't see a definition of `VoidFunction` in this file, where is it coming from?

Reply:
This was a mistake on my part. Originally I had added a helper `cleanup` utility, then thought better of it because nobody's asking for that additional helper and not enough people would use it to be worth the docs.